.github/workflows/equinix_k8s_flow.yml

```diff
@@ -147,4 +218,12 @@ jobs:
           echo "indexers:" >> kubelet-density.yml
           echo " - type: local" >> kubelet-density.yml
           echo "   metricsDirectory: collected-metrics" >> kubelet-density.yml
+          export START_TIME=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
           ../../../kube-burner init -c kubelet-density.yml -u http://localhost:9090 --metrics-profile metrics.yaml
+          export END_TIME=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+          # Retrieve metrics related to Kepler cpu and memory utilization over time
+          curl -G 'http://localhost:9090/api/v1/query_range' \
+            --data-urlencode 'query=sum (rate (container_cpu_usage_seconds_total{namespace="kepler"}[1m])) / sum(machine_cpu_cores) * 100' \
```

Comment:
@jtaleric i found this query to calculate kepler pod cpu utilization from stackoverflow. If you have a better one, please share :D